<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="commit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>410</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>441</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="name_label_2">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="password_label_2">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="confirmPassword_label_2">
        <property name="text">
         <string>Confirm Password </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="sign_label_2">
        <property name="text">
         <string>个性签名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="head_label_2">
        <property name="text">
         <string>头像</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLineEdit" name="name_line_2"/>
      </item>
      <item>
       <widget class="QLineEdit" name="password_line_2"/>
      </item>
      <item>
       <widget class="QLineEdit" name="confirmPassword_line_2"/>
      </item>
      <item>
       <widget class="QLineEdit" name="sign_line_2"/>
      </item>
      <item>
       <widget class="QLineEdit" name="head_line_2"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="tip_label">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>401</y>
     <width>216</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="register_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>              注册</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
